processing/src/main/java/org/apache/druid/query/groupby/epinephelinae/vector/DictionaryBuildingSingleValueStringGroupByVectorColumnSelector.java

```diff
@@ -82,7 +83,7 @@ public int writeKeys(
 
         // Use same ROUGH_OVERHEAD_PER_DICTIONARY_ENTRY as the nonvectorized version; dictionary structure is the same.
         stateFootprintIncrease +=
-            DictionaryBuilding.estimateEntryFootprint((value == null ? 0 : value.length()) * Character.BYTES);
+            DictionaryBuildingUtils.estimateEntryFootprint((value == null ? 0 : value.length()) * Character.BYTES);
```

Comment:
Should we push this stringLogic inside the footPrintMethod or maybe have a wrapper string method inside the same class ?